.gitignore

```diff
@@ -14,3 +14,6 @@
 .settings
 /.apt_generated
 /html
+
+# Idea Files
+/.idea
```

Comment:
Missing newline at end of file.

(I know this .gitignore doesn't really belong to this PR, but I'm fine with it being here as IntellJ's files should also be ignored anyways. So leave it in, but please fix the newline ^^)